dbt_semantic_interfaces/validations/element_const.py

```diff
@@ -51,7 +53,7 @@ def validate_model(model: SemanticManifest) -> Sequence[ValidationIssue]:  # noq
 
     @staticmethod
     def _get_element_name_to_types(
-        model: SemanticManifest,
+        model: PydanticSemanticManifest,
```

Comment:
I got this far before it occurred to me, but wow, we use `model` in a lot of places that should now be `manifest` or `semantic_manifest`

This is a problem for later.